actions/update_changelog.meta.yaml

```diff
@@ -0,0 +1,42 @@
+---
+name: update_changelog
```

Comment:
Because we create git tag (which is more important for release, comparing to changelog), I think we'll need a better naming for this action.